packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -0,0 +1,562 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/foundation.dart';
+import 'package:vector_math/vector_math_64.dart' show Matrix4;
+
+import 'box.dart';
+import 'object.dart';
+import 'viewport.dart';
+import 'viewport_offset.dart';
+
+typedef double _ChildSizingFunction(RenderBox child);
+
+class _ListWheelParentData extends ContainerBoxParentData<RenderBox> { }
+
+/// Render, onto a wheel, a bigger sequential set of objects inside this viewport.
+///
+/// Takes a scrollable set of fixed sized [RenderBox]es and renders them
+/// sequentially along the scrolling axis.
```

Comment:
Since this only works in one direction, down, it would be better for the docs to be explicit about this. Otherwise people will hunt and hunt for ways to make it go up or horizontal.

Reply:
Done